src/lib/providers/CurrencyProvider.tsx

```diff
@@ -29,14 +29,16 @@ const CurrencyProvider: React.FC<CurrencyProviderType> = ({ children }) => {
 	// Poll CoinGecko every 30 seconds
 	const getWildPriceUsd = async () => {
 		const price = await wildTokenPrice();
-		setWildPriceUsd(price || 0);
+		setWildPriceUsd(price);
+		if (price === 0) console.error('$WILD price API call failed');
```

Comment:
Always use brackets when making if statements please.

Reply:
You should be try/catching await lootTokenPrice() to see if it fails, not by checking if the price is zero.